docs/DataProviderList.md

```diff
@@ -46,6 +46,7 @@ If you can't find a Data Provider for your backend below, no worries! [Writing a
 * **[OpenCRUD](https://www.opencrud.org/)**: [weakky/ra-data-opencrud](https://github.com/Weakky/ra-data-opencrud)
 * **[Parse](https://parseplatform.org/)**: [almahdi/ra-data-parse](https://github.com/almahdi/ra-data-parse)
 * **[PostGraphile](https://www.graphile.org/postgraphile/)**: [bowlingx/ra-postgraphile](https://github.com/BowlingX/ra-postgraphile)
+* **[Postgres](https://github.com/mingfang/ra-postgres-dataprovider)
```

Comment:
The syntax isn't correct: the list shows first the name of the backend in bold, then the name of the adapter. Could you please fix it?